package.json

```diff
@@ -2,7 +2,7 @@
   "name": "iterate-root",
   "type": "module",
   "scripts": {
-    "dev": "pnpm -r --reporter-hide-prefix --parallel dev",
+    "dev": "pnpm -r --reporter-hide-prefix --filter \"@iterate-com/os\" --parallel dev",
```

Comment:
hmm does this disable ngrok? note: https://x.com/EastlondonDev/status/1984280151899910514

Reply:
agreed about all that, though I don't think we should touch that stuff in this PR